aspnetcore/blazor/hosting-models.md

```diff
@@ -22,10 +22,7 @@ The principal hosting model for Blazor is running client-side in the browser on
 
 ![Blazor client-side: The Blazor app runs on a UI thread inside the browser.](hosting-models/_static/client-side.png)
 
-To create a Blazor app using the client-side hosting model, use either of the following templates:
-
-* **Blazor (client-side)** ([dotnet new blazor](/dotnet/core/tools/dotnet-new)) &ndash; Deployed as a set of static files.
-* **Blazor (ASP.NET Core Hosted)** ([dotnet new blazorhosted](/dotnet/core/tools/dotnet-new)) &ndash; Hosted by an ASP.NET Core server. The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
+To create a Blazor app using the client-side hosting model, use the **Blazor WebAssembly App** template ([dotnet new blazorwasm](/dotnet/core/tools/dotnet-new)). After selecting the **Blazor WebAssembly App** template, select the **Hosted** option to configure the app to use an ASP.NET Core server backend ([dotnet new blazorwasm --hosted](/dotnet/core/tools/dotnet-new)). The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
```

Comment:
This is for discussion ... I liked it as two paragraphs to make it clearer that the hosted option is just that ... an option. I add a hair more text to make that point clear. This is just for discussion thus far ... I'm not fully married to the exact language. I'm just worried that peeps will get confused about it.

```suggestion
To create a Blazor app using the client-side hosting model, use the **Blazor WebAssembly App** template ([dotnet new blazorwasm](/dotnet/core/tools/dotnet-new)).

After selecting the **Blazor WebAssembly App** template, you have the option of configuring the app to use an ASP.NET Core backend by selecting the **Hosted** option ([dotnet new blazorwasm --hosted](/dotnet/core/tools/dotnet-new)). The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
```

Reply:
I think we can call out the fact that there are 3 versions of Blazor

- Server-Side Blazor (the artist formerly known as Razor Components)
- Web Assembly Blazor
  - Standalone single project template
  - ASP.NET Core Hosted 3 project template

But this is represented by 2 actual templates, one having some additional configuration which is optional

Reply:
Sounds good ... my only concern was that if we run the *hosted* option right on the back of the *non-hosted* option (same paragraph and with the instruction to "After selecting the **Blazor WebAssembly App** template, select ...") that it gets a little confusing and almost sounds like the *hosted* option must be used.

Reply:
sure, defer to @danroth27 on intended language

Reply:
@guardrex 's suggestion looks good to me.

Reply:
I think though that the `--hosted` option shows up as "ASP.NET Core hosted" in VS. Will folks using VS understand what we mean by the "Hosted" option?

Reply:
It's a check box, right? If it is, I can change my suggestion to say ...

> ... backend by selecting the **Hosted** check box when creating the app ([dotnet new blazorwasm --hosted](/dotnet/core/tools/dotnet-new)).

Reply:
```suggestion
To create a Blazor app using the client-side hosting model, use the **Blazor WebAssembly App** template ([dotnet new blazorwasm](/dotnet/core/tools/dotnet-new)).

After selecting the **Blazor WebAssembly App** template, you have the option of configuring the app to use an ASP.NET Core backend by selecting the **Hosted** check box ([dotnet new blazorwasm --hosted](/dotnet/core/tools/dotnet-new)). The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
```

Reply:
@guardrex Yeah, it's a checkbox, but the label for the checkbox is "ASP.NET Core hosted"

Reply:
Ah ... gotcha. I'll update that suggestion :point_up:.

I really, **_really_** need to get Aquent to send me a laptop. 🤦‍♂

I'll do that ... I'll send them a message now.

Note to self on next desktop build ... **_MASSIVE STORAGE! Plz!_**. lol